arrow/src/test/java/org/apache/calcite/adapter/arrow/ArrowData.java

```diff
@@ -237,35 +239,38 @@ public void writeArrowDataType(File file) throws IOException {
       vectorSchemaRoot.setRowCount(numRows);
       for (Field field : vectorSchemaRoot.getSchema().getFields()) {
         FieldVector vector = vectorSchemaRoot.getVector(field.getName());
-        switch (vector.getMinorType()) {
-        case TINYINT:
+        switch (field.getName()) {
```

Comment:
why change these?

Reply:
Sorry for taking so long to reply. 
This is because I set two fields for decimal type, one with scale 2 and one with scale 3. Since there are two decimal types, it is no longer appropriate to distinguish based on data type.

Reply:
So, this is just test code. Can you change the file name to ArrowTestData?

Reply:
@mihaibudiu thanks, done